portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +25,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  public ArrayList<String> randomFacts;
+
+  @Override
+  public void init() {
+      randomFacts = new ArrayList<String>();
+      randomFacts.add("Hello Malachi");
+      randomFacts.add("Red");
+      randomFacts.add("Computer Science and Cognitive Science");
+  }
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    init();
+    response.setContentType("application/json;");
+    String json = convertToJson(randomFacts);
+    response.getWriter().println(json);
+  }
+
+  public String convertToJson(ArrayList<String> randomFacts) {
```

Comment:
Does the convertToJson method need to be public? If not let's make it private to restrict visibility as much as we can.

Reply:
Done.